meta-oe/recipes-support/librdkafka/librdkafka/0001_fix_absolute_path_usage.patch

```diff
@@ -0,0 +1,24 @@
+From dd06894a786edf4bea11dace50d7380b89dfaba5 Mon Sep 17 00:00:00 2001
+From: hasan.men <hasan.men@bosphorusiss.com>
+Date: Sun, 4 Apr 2021 17:20:32 +0200
+Subject: [PATCH] fix absolute path problem
+
```

Comment:
please submit this patch upstream as well. If its not worthy of upstream then added the status to patch comment so we know we have to carry it for a reason.

Reply:
@kraj I've already pushed. Just waiting for response. Thanks 

https://github.com/edenhill/librdkafka/pull/3327